samza-core/src/main/java/org/apache/samza/execution/JobGraph.java

```diff
@@ -248,10 +248,10 @@ void validate() {
   }
 
   /**
-   * Validate the sources should have indegree being 0 and outdegree greater than 0
+   * Validate the input streams should have indegree being 0 and outdegree greater than 0
    */
   private void validateSources() {
```

Comment:
s/validateSources/validateInputStreams. 
Just to keep it consistent with naming in rest of the classes.

Reply:
Absolutely. Thanks again.